grafana/dashboards/brigade.json

```diff
@@ -207,7 +208,7 @@
             },
             "lineInterpolation": "linear",
             "lineWidth": 1,
-            "pointSize": 5,
+            "pointSize": 4,
```

Comment:
What does point size do?

Reply:
It dictates the size of the plot points on the graph. With a 2s frequency on the graph, a pointSize of 5 is too big and the dots kind of mesh together, so I changed it to 4

Reply:
I think I'm missing something. I don't see any dots to begin with.

![Screen Shot 2021-08-01 at 6 40 39 PM](https://user-images.githubusercontent.com/1821014/127787406-4b921ba8-4262-431f-8975-994050f7a392.png)

Reply:
When there are too many dots in one time series, the dots disappear. Try setting the time range to the last 30 minutes or less, and you will see the dots. I made it smaller because even when they appeared, they are too big and almost overlap

Reply:
Made the time range smaller. See it now. Thanks.